### PR TITLE
React Styled Components - Demo

### PR DESCRIPTION
**Pull request by aclaes**
### 🔎 Review

Review via [🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/react-styled-components_demo/sessions/react-styled-components/demo-start?file=/README.md) or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/react-styled-components_demo/sessions/react-styled-components/demo-start -i
```

### ✨ Copyable Markdown for Curriculum

````md
Use this demo on
[🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/main/sessions/react-styled-components/demo-start?file=/README.md)
or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/main/sessions/react-styled-components/demo-start -i
```
````
### 🔎 Review

Review via [🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/react-styled-components_demo/sessions/react-styled-components/demo-end?file=/README.md) or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/react-styled-components_demo/sessions/react-styled-components/demo-end -i
```

### ✨ Copyable Markdown for Curriculum

````md
You can check out the final version of this demo on
[🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/main/sessions/react-styled-components/demo-end?file=/README.md)
or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/main/sessions/react-styled-components/demo-end -i
```
````

refers to neuefische/web-curriculum-new-format#210

### CODE REVIEW

File: sessions/react-styled-components/demo-end/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          title="It worked"
+          message="You have done it!"
+          variant="success"
+        />
+        <InfoBox
+          title="Error"
+          message="Something went wrong!"
+          variant="error"
+        />
+      </StyledContainer>
```
[the remark this answers is not in the document]
Comment (aclaes) [reply sessions/react-styled-components/demo-end/pages/index.js]: I updated the naming convention in the demo

File: sessions/react-styled-components/demo-end/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import InfoBox from "../components/InfoBox";
+import StyledButton from "../components/StyledButton";
+import StyledContainer from "../components/StyledContainer";
+import StyledDangerButton from "../components/StyledDangerButton";
+import StyledLink from "../components/StyledLink";
```
[the remark this answers is not in the document]
Comment (aclaes) [reply sessions/react-styled-components/demo-end/pages/index.js]: I updated the naming convention in the demo

File: sessions/react-styled-components/demo-end/components/StyledDangerButton.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+import StyledButton from "./StyledButton";
+
+export default styled(StyledButton)`
```
[the remark this answers is not in the document]
Comment (aclaes) [reply sessions/react-styled-components/demo-end/components/StyledDangerButton.js]: I updated the naming convention in the demo

File: sessions/react-styled-components/demo-end/components/StyledDangerButton.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+import StyledButton from "./StyledButton";
```
[the remark this answers is not in the document]
Comment (aclaes) [reply sessions/react-styled-components/demo-end/components/StyledDangerButton.js]: I updated the naming convention in the demo

File: sessions/react-styled-components/demo-end/components/StyledDangerButton.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
```
[the remark this answers is not in the document]
Comment (aclaes) [reply sessions/react-styled-components/demo-end/components/StyledDangerButton.js]: This component was remove from the demo

File: sessions/react-styled-components/demo-end/components/StyledButton.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
```
[the remark this answers is not in the document]
Comment (aclaes) [reply sessions/react-styled-components/demo-end/components/StyledButton.js]: I updated the naming convention in the demo

File: sessions/react-styled-components/demo-start/README.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# React Styled Components: Demo Start
+
+## Notes
+
+- This is the start version of the demo
```
Comment (CJAkkuec) [inline sessions/react-styled-components/demo-start/README.md:5]:
```suggestion
- This is the starter version of the demo
```

**Review by CJAkkuec — APPROVED** [review]
LGTM, save for one minor little thingie :)

**Pull request merged**